front/src/Api/iframe/room.ts

```diff
@@ -113,7 +113,7 @@ class WorkadventureRoomCommands extends IframeApiContribution<WorkadventureRoomC
                 'layerName': layerName,
                 'propertyName': propertyName,
                 'propertyValue': propertyValue,
-            } as SetPropertyEvent
```

Comment:
i mean you dont need to remove them all just saying its not completely neccessary :D